### DISCUSSION TITLE
Error! No existing credentials found.

### DISCUSSION THREAD

**Discussion by ivenuss** [vercel/community#582] [Help]
Hey, I am getting an error when I try to deploy an app to production. I've tried to run `vercel login` and also pass `--token`, but I am getting the same error.

```
Error! No existing credentials found. Please run `vercel login` or pass "--token"`
Learn More: https://err.sh/vercel/no-credentials-found
Error: Command "vercel build" exited with 1
```

**Comment by samcx** [comment]
@ivenuss Have you tried using the latest Vercel CLI to resolve this issue?

**Comment by njam** [comment]
Cross-posting from https://github.com/vercel/vercel/discussions/7838#discussioncomment-2865196 :

> As a workaround I found that `vercel deploy --force` avoids this problem.

After deploying with `vercel deploy --force` _once_ the problem is gone for me.
